ITEM.tsv

```diff
@@ -895,21 +895,21 @@ ITEM_20150317_000894	Dratt Staff
 ITEM_20150317_000895	Aston Staff 
 ITEM_20150317_000896	{memo X}Devi Staff 
 ITEM_20150317_000897	{memo X}Prima Staff 
-ITEM_20150317_000898	Candle Staff 
-ITEM_20150317_000899	The fire at the tip may go out, but it won't be because it ran out of fuel. 
-ITEM_20150317_000900	Audra 
-ITEM_20150317_000901	Some say that the Cryomancer Master jokingly told the Pyromancer Master to challenge him with this staff using ice magic while he was barehanded. 
-ITEM_20150317_000902	Sunflower 
-ITEM_20150317_000903	In the recipe of this weapon, instructions are written on how to obtain help from Goddess Saule and Goddess Gabija during the crafting process. 
-ITEM_20150317_000904	Maledic 
-ITEM_20150317_000905	Legends say that the original Maledic is in the demonic dimension and that the imitations seen in this world are only projected illusions. 
-ITEM_20150317_000906	Maledoom 
-ITEM_20150317_000907	A group of wizards succeeded in crafting the more powerful Maledoom while they were developing a crafting process for Maledic. 
-ITEM_20150317_000908	Chapparition Staff 
-ITEM_20150317_000909	Arca Staff 
-ITEM_20150317_000910	According to one tale, Agailla Flurry used to carry this kind of staff when she was constructing the Mage Tower. 
-ITEM_20150317_000911	Raganos Horn 
-ITEM_20150317_000912	Hauberk, a Demon King sealed a long time ago, revealed the secrets of the Demon world. Agailla Flurry created this recipe based on that revealed information. 
+ITEM_20150317_000898	$Candle Staff 
+ITEM_20150317_000899	$The fire at the tip may go out, but it won't be due to it running out of fuel. 
+ITEM_20150317_000900	$Audra 
+ITEM_20150317_000901	$Some say that the Cryomancer Master jokingly challenged the Pyromancer Master to fight him with this staff using ice magic while he was barehanded. 
+ITEM_20150317_000902	$Sunflower 
+ITEM_20150317_000903	$In the recipe for this weapon, instructions are written on how to obtain help from the Goddesses Saule and Gabija during the crafting process. 
+ITEM_20150317_000904	$Maledic 
+ITEM_20150317_000905	$Legends say that the original Maledic is still in the demonic dimension, and that the imitations seen in this world are only projected illusions. 
+ITEM_20150317_000906	$Maledoom 
+ITEM_20150317_000907	$A group of wizards succeeded in crafting the powerful Maledoom while they were developing a crafting process for Maledic. 
+ITEM_20150317_000908	$Chapparition Staff 
+ITEM_20150317_000909	$Arca Staff 
+ITEM_20150317_000910	$According to one tale, Agailla Flurry used to carry this kind of staff when she was constructing the Mage Tower. 
+ITEM_20150317_000911	$Raganos Horn 
+ITEM_20150317_000912	$Hauberk, a Demon King sealed ages ago, revealed the secrets of the Demon world. Agailla Flurry created this recipe using this revealed information. 
```

Comment:
@imcgames Hauberk is a Demon Lord, but in this description it says Demon King (마왕) - Should this be changed to Demon Lord, to avoid inconsistencies? Right now it's inconsistent.